### DISCUSSION TITLE
I can't get my account approved, although I've been using GH Classroom for years

### DISCUSSION THREAD

**Discussion by eblanton** [community/community#43769] [GitHub Education] [answered]
I'm faculty at SUNY at Buffalo.  I've been using GH Classroom and GH Education orgs for years.  Now when I submit my new documents (?????), I get:

> Your application cannot be reviewed until you fix the following:
> * You are significantly more likely to be verified if you have completed your GitHub user profile with your full name and a short bio.

I already have these things.  I tried to file a ticket with GH about some other problems, and they closed it without response and redirected me here.  If any community members know how to fix this, I guess I'm listening?  I mostly see unanswered posts with similar problems, so...

**Comment by cmumatt** [comment]
@eblanton Hard same, except as a student.

**Comment by eblanton** [comment]
I regret the lack of support for education shown here.  My semester starts soon, and I cannot get my orgs blessed for use with GitHub Classroom.

**Comment by eblanton** [comment, marked as answer]
I tried again, changing absolutely nothing, and the form went through immediately.  I don't know if it was a bug on the server side, or a browser state problem, or things just requiring time to propagate, or what.  I of course never received any support from GitHub on this issue.
